data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
+  "shelfmark": "Sinai Syriac 2A",
+  "collection": "Old Collection",
+  "repository": "https://sinaimanuscripts.library.ucla.edu/place/1",
+  "form": "Codex",
+  "binding": {
+    "exists": true,
+    "type": "",
+    "note": ["Relatively recent; leather on wood showing signs of repair; metal studs (some missing) on front and back and along edges; remnants of clasps (four total, two for edge and one each at top and bottom)"],
+    "features": []
+  },
+  "extent": ["f. 1-180"],
+  "weight": "3788.3 g",
+  "dimensions": [{
+    "type": "object",
+    "value": "328 x 255 x 97 mm"
+  }],
+  "condition": {
+    "note": [],
+    "features": []
+  },
+  "provenance": [{
+    "type": "acquisition",
+    "note": ["Created and remained in Saint Catherine's Monastery"],
+    "features": [],
+    "source": []
+  }],
+  "paratexts": [],
+  "associated_persons": [],
```

Comment:
These fields can stay, if needed, but will likely be plugged in primarily under the paratexts and events. But if we need a generic one that doesn't go with a paratext or event it could go here.